tests/lac/utilities_01.output

```diff
@@ -0,0 +1,21 @@
+
+DEAL::Size 31 Unknowns 30
+DEAL::
+DEAL::4
+DEAL::Lanczos 4.83338
+DEAL::CG 3.72123
```

Comment:
neat, getting the two estimates side by side.

Would it be worth pointing out in the documentation of the new function that using CG is an alternative, and maybe in the documentation of CG that this function is an alternative?